app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  after_save :update_posts_counter
+
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def last_five_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] To return the _most 5 recent _ comments in order, our fetch request implementation should be similar to:
```ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
```
Kindly ensure to improve upon the same.